example/lib/main.dart

```diff
@@ -17,6 +17,7 @@ class _MyAppState extends State<MyApp> {
   static const String iosChannelKey = "eyJzZXR0aW5nc191cmwiOiJodHRwczovL2hhbmFtaWhlbHAuemVuZGVzay5jb20vbW9iaWxlX3Nka19hcGkvc2V0dGluZ3MvMDFGR1BGVFQ1Q1hFRjdRWVkwUkg2R0JYS0MuanNvbiJ9";
 
   final List<String> channelMessages = [];
+  int unreadMessageCount =0;
```

Comment:
format code

Reply:
Fixed in 20b802ed8f51d865d4df359d352885757e5c5a9c